run_azcopy/run_azcopy.sh

```diff
@@ -21,4 +21,7 @@ AZCOPY_OPTIONS=("--recursive" "--overwrite" "true")
 if [[ "$(echo "$PUT_MD5" | tr '[:upper:]' '[:lower:]')" == 'true' ]]; then
   AZCOPY_OPTIONS+=("--put-md5")
 fi;
-./azcopy copy "$SOURCE" "$TARGET" "${AZCOPY_OPTIONS[@]}"
+if [[ "$(echo "$DELETE_DESTINATION" | tr '[:upper:]' '[:lower:]')" == 'true' ]]; then
```

Comment:
Did you test? Looks like, this PR implies that input `delete_destination` is implicitly mapped on environment variable `DELETE_DESTINATION`, but I didn't find any mentions of that in documentation.

Not sure if it will work. In other actions, inputs are mapped on environment variables [explicitly](https://github.com/SneaksAndData/github-actions/blob/897837ea19e3b1986b4fcb3ef2985af951c87d58/run_azcopy/action.yaml#L31).

Reply:
Now explicitly mapped :)